packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -196,6 +196,15 @@ function BlockSelectionButton( { clientId, rootClientId, blockElement } ) {
 				if ( navigateDown ) {
 					nextTabbable = focus.tabbable.findNext( blockElement );
 
+					if ( null === getNextBlockClientId( clientId ) ) {
```

Comment:
Couldn't think of a better way to do this. Open to feedback on how to better focus the Settings Sidebar. This code was great before these new Blocks, it just doesn't work with the more dynamic Blocks.